### DISCUSSION TITLE
Debug does not appear in phpinfo

### DISCUSSION THREAD

**Discussion by JAyenGreen** [shivammathur/homebrew-php#1018] [Q&A / Help] [answered]
I have shivammathur php and extensions/xdebug@8.1 installed. I have done the link and restarted the service.

The php version is confirmed in phpinfo:

```
System => Darwin Jeffs-Air.local 21.3.0 Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_ARM64_T8101 arm64
Build Date => Aug  4 2022 03:58:04
Build System => Darwin b44c11f5-910a-4e3b-b6ea-3c543834efe5 21.5.0 Darwin Kernel Version 21.5.0: Tue Apr 26 21:08:29 PDT 2022; root:xnu-8020.121.3~4/RELEASE_ARM64_T8101 arm64
Configure Command =>  './configure'  '--prefix=/opt/homebrew/Cellar/php-debug/8.1.9' '--localstatedir=/opt/homebrew/var' '--sysconfdir=/opt/homebrew/etc/php/8.1-debug' '--with-config-file-path=/opt/homebrew/etc/php/8.1-debug'

```
but there is no mention in the extensions of xdebug, and attempting to debug a script in PhpStorm has it indicate that the interpreter does not have xdebug installed.

**Comment by shivammathur** [comment, marked as answer]
@JAyenGreen 

`shivammathur/extensions` tap does not have debug versions of extensions. 

Please install `php` instead of `php-debug` if you want to use extensions from the tap. 

Or if you need the debug build of PHP please use PECL for now to install extensions.

**Reply by JAyenGreen** [reply]
Ok. I did:
brew remove shivammathur/php/php-debug
brew install shivammathur/php/php@8.1

brew list shows:

shivammathur/extensions/xdebug@8.1
shivammathur/php/php

brew link --overwrite --force php@8.1 (already linked)
brew services restart php
php -a
phpinfo();

xdebug is still not listed.

**Reply by shivammathur** [reply]
I ran a test and it is working correctly
https://github.com/shivammathur/test-setup-php/actions/runs/2890467133/workflow
https://github.com/shivammathur/test-setup-php/runs/7921188338?check_suite_focus=true#step:4:706

Please try re-installing xdebug and check again.
```
brew reinstall shivammathur/extensions/xdebug@8.1
```

**Reply by JAyenGreen** [reply]
Magic!
